addon-test-support/run-options.ts

```diff
@@ -0,0 +1,32 @@
+import { RunOptions } from 'axe-core';
+import QUnit from 'qunit';
+import { getContext } from '@ember/test-helpers';
+
+let _runOptions = new WeakMap<object, RunOptions>();
+
+/**
+ * Sets run options specific to a test.
+ *
+ * @param options Axe {RunOptions} to be provided to the audit helper.
+ */
+export function setRunOptions(options: RunOptions = {}) {
+  _runOptions.set(getContext(), options);
+}
+
+/**
+ * Gets run options specific to a test.
+ *
+ * @param context Test context object, accessed using `@ember/test-helpers` `getContext` function.
+ */
+export function getRunOptions(context: object) {
+  return _runOptions.get(context);
+}
+
+/**
+ * Sets up the cleanup of the run options for a specific test.
+ */
+export function setupRunOptions() {
+  QUnit.testDone(function () {
```

Comment:
I don't really want to bake in `QUnit` semantics here

Reply:
Ya I'm with you. I'm writing this while we're speaking about using the destroyables APIs. I agree. :)